src/cocaine-app/sync/__init__.py

```diff
@@ -20,8 +20,11 @@
 
 
 def encode_dict(params):
```

Comment:
There is such function in `src/cocaine-app/cache_transport/__init__.py`, why do you need both?